app/javascript/src/controller_pages.js

```diff
@@ -380,18 +386,6 @@ function enhanceQuestions(view) {
         }
       },
 
-      onItemRemoveConfirmation: function(item) {
-        // @item (EditableComponentItem) Item to be deleted.
-        // Runs before onItemRemove when removing an editable Collection item.
-        // Currently not used but added for future option and consistency
-        // with onItemAdd (provides an opportunity for clean up).
-        view.dialogConfirmationDelete.open({
-          heading: view.text.dialogs.heading_delete_option.replace(/%{option label}/, item._elements.label.$node.text()),
```

Comment:
Ha! Looks like I had muddied the water on my other comment about using `#{something}` form of lookup label, as this one used `%{something_else}` to identify the swappable part. Maybe I had some 'this one is for templates' and 'this one is for JS files' kind of thinking going on. Overly complex. Who cares now, since the lines are being removed :-)